github/Branch.py

```diff
@@ -130,6 +130,7 @@ def edit_protection(
         self,
         strict: Opt[bool] = NotSet,
         contexts: Opt[list[str]] = NotSet,
+        checks: Opt[list[Union[str, Tuple[str, int]]]] = NotSet,
```

Comment:
Please add at the end of the signature, or you may break user code.

Reply:
thanks, didn't think about that - fixed!